KPP/fullchem/gckpp_Rates.F90

```diff
@@ -77,7 +77,7 @@ SUBROUTINE Update_RCONST ( )
   RCONST(13) = (GCARR_ac(2.45d-12,-1775.0d0))
   RCONST(14) = (GC_RO2NO_B1_ac(2.80d-12,300.0d0))
   RCONST(15) = (GC_RO2NO_A1_ac(2.80d-12,300.0d0))
-  RCONST(16) = (GCARR_abc(4.10E-13,0.0E+00,750.0))
+  RCONST(16) = (GCARR_abc(4.10d-13,0.0d+00,750.0d0))
```

Comment:
This type of bug fix will go into the standard model.

Reply:
Great!